update.php

```diff
@@ -500,6 +500,43 @@
     }
 }
 
+// 20150401 Add mail method to database (SMTP/sendmail)
+$sql = "SELECT COUNT(*) AS confcnt FROM config";
+$req = $pdo->prepare($sql);
+$req->execute();
+$confcnt = $req->fetch(PDO::FETCH_ASSOC);
+
+if ($confcnt['confcnt'] < 20) {
+    $mail_method = 'sendmail';
+    // check if an smtp server was set
+    $sql = "SELECT * FROM config";
+    $req = $pdo->prepare($sql);
+    $req->execute();
+    $config_items = [];
+    while ($show = $req->fetch()) {
+        array_push($config_items, $show);
+    }
+
+    if ($config_items['smtp_address'] !== '') {
+        $mail_method = 'smtp';
+        $smtp_username = filter_var($config_items['smtp_username'], FILTER_VALIDATE_EMAIL);
```

Comment:
Great :) I'm still testing my code against different setups, so certain things might still be subject to change.